CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ customers cannot upgrade their bootloader, its changes are recorded separately.
 ## Firmware
 
 ### [Unreleased]
+- Updated BTC/LTC amount formatting, avoiding to trim trailing zeroes for better clarity
```

Comment:
i think it's clearer to say it directly like 'Updated BTC/LTC amount formatting to display the trailing zeroes`.